### DISCUSSION TITLE
Bienvenida gente de la comision 144

### DISCUSSION THREAD

**Discussion by GabrielAguilar2023** [GabrielAguilar2023/Comision144#1] [Announcements]
Buenos Dias!!! Mi nombre es Gabriel Aguilar y este sería el comienzo de mi primer trabajo en equipo...

**Comment by JavierMunho** [comment]
Buenas tardes soy Javier Munhó y te agradezco, la invitación a formar parte de este grupo.

**Reply by GabrielAguilar2023** [reply]
Hola Javier, un gusto!... Esta bueno empezar a probar estas herramientas, nuevas para mi, pero a la vez re-utiles para programar en grupo. Yo, particularmente sé algo de programacion, pero nunca trabaje en grupo, así que sera una experiencia nueva para mi...

**Comment by juicich** [comment]
Hola Gabriel !! Soy Javier Uicich. Trataré de colaborar en el grupo. Una experiencia nueva para mi. Slds!!

**Reply by GabrielAguilar2023** [reply]
Hola Javier! Espero que todos podamos aprender algo y saquemos cosas positivas de esta experiencia de trabajar en grupo a traves de esta herramienta. No soy experto, ni mucho menos en el manejo de github, asi que entre todos, espero podamos aprender a usar todo esto y de paso darnos una mano en el desarrollo del trabajo final para aprobar el curso, Asique... Bienvenido al grupo y estamos en contacto.

**Comment by Erica1975** [comment]
Hola!! Soy Erica! Empezando con esto!!!

**Reply by GabrielAguilar2023** [reply]
Hola Erica! Muy bien por empezar...  Queres que te agregue como colaboradora para que puedas subir y modificar cosas y asi poder aprender a usar las diferentes secciones de ese entorno?

**Comment by Erica1975** [comment]
Si por favor, estoy media atrasada porque estoy con otro curso de EGG...y
rindo el Lunes...pero ya me pongo el finde con esto!!!

**Reply by GabrielAguilar2023** [reply]
Dale! Ya te agregué. Ahora te debería llegar un email, lo aceptas y ya podrías empezar... Ah! y mucha suerte con tu examen.

**Comment by Erica1975** [comment]
GRACIAS!
